pkg/porter/testdata/runs/expected-output.json

```diff
@@ -0,0 +1,18 @@
+[
```

Comment:
Did you forgot to check-in the new test that uses these files? I don't see where they are being used.

Looking at the test data here, I noticed that `action` and `status` are not populated, and that there is not a testdata file for plain text output. So that may be something to look at quick when you add that test.

Reply:
Sounds good, let me know when this is ready to review again!

Reply:
Ready to review again